pkg/projects/dir.targets

```diff
@@ -18,11 +18,11 @@
     <!-- Update project.json template -->
     <MakeDir Directories="$([System.IO.Path]::GetDirectoryName('$(ProjectJson)'))" />
 
-    <ReplaceFileContents
-      InputFile="project.json.template"
-      DestinationFile="$(ProjectJson)"
-      ReplacementPatterns="{RID};{TFM}"
-      ReplacementStrings="$(TargetRid);$(NuGetTargetMoniker)" />
+    <WriteLinesToFile
```

Comment:
Another option here:

```
    <ItemGroup>
      <_ProjectJsonReplacementValues Include="{RID}">
        <ReplacementString>$(NuGetRuntimeIdentifier)</ReplacementString>
      </_ProjectJsonReplacementValues>
      <_ProjectJsonReplacementValues Include="{TFM}">
        <ReplacementString>$(NuGetTargetMoniker)</ReplacementString>
      </_ProjectJsonReplacementValues>
    </ItemGroup>

    <ReplaceFileContents
      InputFile="project.json.template"
      DestinationFile="$(ProjectJson)"
      ReplacementItems="@(_ProjectJsonReplacementValues)" />
```

It's a little more "MSBuildy", and allows it to be extended easily.